src/data/components/Greetings/Greetings.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+function Greetings(props) {
+    let {lang, children} = props;
+    switch(lang){
+        case 'de':
+            lang = 'Hallo';
+            break;
+        case 'en':
+            lang = 'Hello';
+            break;
+        case 'es':
+            lang = 'Hola';
+            break;
+        case 'fr':
+            lang ='Bonjour';
+            break;
+        default:
+            return 'N/A';
+            break;
+    }   
+  return (
+      <div className='Greetings'> {lang} {children} </div>
+  )
+}
+
+export default Greetings
```

Comment:
Here, you should not reassign a value that you get from props. You should either create a separate variable or just return plain HTML